clouddriver-ecs/src/integration/resources/testoperations/createServerGroup-input-EC2-targetGroupMappings-existingService.json

```diff
@@ -0,0 +1,37 @@
+{
+  "account": "ecs-account",
+  "application": "ecs",
+  "availabilityZones": {
+    "us-west-2": [
+      "us-west-2a",
+      "us-west-2c"
+    ]
+  },
+  "capacity": {
+    "desired": 1,
+    "max": 1,
+    "min": 1
+  },
+  "cloudProvider": "ecs",
+  "computeUnits": 256,
+  "credentials": "ecs-account",
+  "dockerImageAddress": "nginx",
+  "ecsClusterName": "integInputEC2TgMappingsExistingService-cluster",
+  "launchType": "EC2",
+  "networkMode": "bridge",
+  "placementStrategySequence": [],
+  "reservedMemory": 512,
+  "stack": "integInputEC2TgMappingsExistingServiceStack",
+  "freeFormDetails" : "detailTest",
```

Comment:
this field is not required and not consistently used between tests. Can we remove it?